scripts/resources/coredns-cm.yaml

```diff
@@ -1,3 +1,3 @@
-    supercluster.local:53 {
+    clusterset.local:53 {
```

Comment:
This may create a mis-match as helm configures the Lighthouse coreDNS authoritative for supercluster.local but only the requests to clusterset will be forwarded to it  by default coredns with this config,  Two entries as below should solve it.

    clusterset.local:53 {
        forward . dnsip
    }
    supercluster.local:53 {
        forward . dnsip
    }

Reply:
Agree. I think we should do it in two steps, even with helm and operator:
1. Add `clusterset`. We'll have both `supercluster` and `clusterset` for 0.6.0
2. Remove 'supercluster`.

This will allow all users and test folks to transition to clusterset without breaking anything.